noetic/distribution.yaml

```diff
@@ -4409,6 +4409,22 @@ repositories:
       url: https://github.com/MITRE/kvh_geo_fog_3d.git
       version: noetic-devel
     status: maintained
+  l3cam_ros:
+    doc:
+      type: git
+      url: https://github.com/beamaginelidar/l3cam_ros.git
+      version: master
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/beamaginelidar/l3cam_ros-release.git
```

Comment:
Seems to be missing, got a 404 on this url

Reply:
When copy-pasting the url into mozilla firefox it is found. I don't really understand why it got 404.
![Screenshot 2023-03-28 09:58:20](https://user-images.githubusercontent.com/102601124/228168695-a2fbce78-ee70-4f41-8d1f-562b823b8a1b.png)

Reply:
@adrisubi by that screenshot it appears that this release repository is private. The release repository must be publicly visible in order for our builds to use it.

Reply:
Oh my bad, didn't notice it was private, now it's public. Tank you